src/Hamelin.Runtimes.GitHubActions/Logging/GitHubActionsConsoleFormatter.cs

```diff
@@ -37,10 +39,16 @@ TextWriter textWriter
         }
 
         string message = logEntry.Formatter.Invoke(logEntry.State, logEntry.Exception);
```

Comment:
Should we apply the same alteration to the log message? In case it contains a newline too.

```suggestion
        string message = logEntry.Formatter.Invoke(logEntry.State, logEntry.Exception)
            .Replace("\r", "")
            .Replace("\n", UrlEncodedNewLine);
```

Reply:
Done!